src/com/pk_projects/to_mst/converter/XMLGraphConverter.java

```diff
@@ -3,14 +3,100 @@
 import com.pk_projects.to_mst.structure.Graph;
 
 import java.io.File;
+import javax.xml.parsers.*;
+import javax.xml.transform.*;
+import javax.xml.transform.dom.*;
+import javax.xml.transform.stream.*;
+import org.w3c.dom.*;
 
 public class XMLGraphConverter extends GraphConverter {
 
+
+    private Graph graph;
```

Comment:
Weź to popraw ... XMLGrapConverter dziedzicy z Graph więc jako tako ta klasa jest grafem tylko że z dodatkową metodą getConverted.